i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md

```diff
@@ -1,65 +1,65 @@
 ---
 id: applications-protocol-snmp
-title: SNMP Protocol
+title: Generic SNMP
 ---
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-## Contenu du Pack
+## Contenu du pack
 
 ### Modèles
 
 Le connecteur de supervision **Generic SNMP** apporte 2 modèles d'hôte :
 
-* App-Protocol-SNMP
-* App-Protocol-SNMP-Only
+* **App-Protocol-SNMP-custom**
+* **App-Protocol-SNMP-Only-custom**
 
-Le connecteur apporte les modèles de service suivants (classés par modèle d'hôte):
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
 
 <Tabs groupId="sync">
-<TabItem value="App-Protocol-SNMP" label="App-Protocol-SNMP">
+<TabItem value="App-Protocol-SNMP-custom" label="App-Protocol-SNMP-custom">
 
 | Alias | Modèle de service | Description |
 |:------|:------------------|:------------|
```

Comment:
you can't have an empty table here, you need to replace it by a sentence saying that no service templaes are linked to this host template